harmonized biomass properties and updated biochar pyrolys

BertoGBG opened the pull request:
Closes # (if applicable).

## Changes proposed in this Pull Request
1) update of biochar pyrolysis:
  - units aligned with multilink structure (i.e. per tCO2 sequestred)
  - introduced biomass drying before pyrolysis
2) introduced function "biomass_properties" where LHV, C%, moisture and other parameters are set.
the function is used to: 1) calculate the biomass carbon intensity and it fixes a small bug, 2) input for biochar pyrolysis

## Checklist

- [ OK] Code changes are sufficiently documented; i.e. new functions contain docstrings and further explanations may be given in `doc`.
- [OK ] Data source for new technologies is clearly stated.
- [ ] Newly introduced dependencies are added to `environment.yaml` (if applicable).
- [ ] A note for the release notes `doc/release_notes.rst` of the upcoming release is included.
- [ ] I consent to the release of this PR's code under the GPLv3 license.

BertoGBG commented:
suggester reviewer : @martavp

martavp commented:
Hi @BertoGBG 

To ease the review, Could you take out the changes that are not related to biochar?
For example, the perennial data in manual_inputs.

Are the changes in values for BioSNG, BtL, biomass-to-methanol, electrobiofuels (i.e. other technologies not related to biochar) intended?

BertoGBG commented:
> Hi @BertoGBG
> 
> To ease the review, Could you take out the changes that are not related to biochar? For example, the perennial data in manual_inputs.
> 
> Are the changes in values for BioSNG, BtL, biomass-to-methanol, electrobiofuels (i.e. other technologies not related to biochar) intended?

Hej Marta

the perennials can be removed. I already explained why it happned. 
the other changes are due to the harmonization of solid biomass properties throughout technology-data. Which effects aslo other functions using the it. I fixed the moisture content for solid biomass, which effects the actual LHV. I fixed it to make it coherent with the energy potentials form JRC report. The C content and LHV of dry biomass are the same that previously used. But the actual LVH was inconsistend throughout the technology-data repository, probably because different sources were used every time. Now ti is coherent but it generated small changes in the output files.
